### ISSUE TITLE
Automatically degrade to i386 on x86_64

### ISSUE THREAD

**Issue by hadess** [flatpak/flatpak#136]
Finally tried what was in #124. 

```
$ flatpak --user remote-ls tutorial-repo
<crickets>
$ flatpak --user install tutorial-repo com.wolfire.Lugaru__The_Rabbit_s_Foot
error: Can't find com.wolfire.Lugaru__The_Rabbit_s_Foot in remote tutorial-repo
$ flatpak --user --arch=i386 install tutorial-repo com.wolfire.Lugaru__The_Rabbit_s_Foot
$ # Yay! worked
```

The binary doesn't exist for x86_64. As all i386 applications should be work on x86_64, when an x86_64 version doesn't exist, the i386 should be tried.

(command-line completion only found the bundle when I forced the arch)

**Comment by hadess** [comment]
Is that fixed in 640bf9e69c726b42e8af398fecc1b3190923e4a8 ?

**Comment by alexlarsson** [comment]
Yes.
